src/cascadia/UIHelpers/Converters.cpp

```diff
@@ -18,6 +18,11 @@ namespace winrt::Microsoft::Terminal::UI::implementation
         return value ? winrt::Windows::UI::Xaml::Visibility::Collapsed : winrt::Windows::UI::Xaml::Visibility::Visible;
     }
 
+    winrt::Windows::UI::Xaml::Visibility Converters::BooleanToVisibility(bool value)
```

Comment:
It doesn't seem like anyone is using these now - can we just back out the UIHelpers changes?